bundle-workflow/python/build.py

```diff
@@ -16,8 +16,8 @@
 
 args = BuildArgs()
 
-component_scripts_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../scripts/bundle-build/components')
-default_scripts_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../scripts/bundle-build/standard-gradle-build')
+component_scripts_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../scripts/components')
+default_scripts_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../scripts/default')
 script_finder = ScriptFinder(component_scripts_path, default_scripts_path)
```

Comment:
Did you forget to change this to use the no-arg ScriptFinder?

Reply:
Yup, thx.